src/js/SubmissionContainer.jsx

```diff
@@ -33,6 +33,26 @@ var SubmissionContainer = React.createClass({
     }
   },
 
+  toggleIRSCheck: function(e){
+    // should make an API call here to POST verification
+    // and if everything is ok it would return the new status
+    if (e.target.checked) {
```

Comment:
I think state should flow entirely from the backend, meaning that in the API call here, the callback needs to verify that the status returned is equal to the state set below. I think setting the state outside of the callback is fine for usability reasons (immediately move to signature component rather than have to wait for callback) and since the backend will pretty much just be noting that a the check box was checked and advancing the state. Do note that a check will need to exist in the callback when it's implemented here. I'm going to ticket something here.